src/modules/hellohook.c

```diff
@@ -44,7 +44,7 @@ void clientChangeCallback(RedisModuleCtx *ctx, RedisModuleEvent e, uint64_t sub,
     REDISMODULE_NOT_USED(e);
 
     RedisModuleClientInfo *ci = data;
-    printf("Client %s event for client #%llu %s:%d\n",
+    printf("Client %s event for client #%lu %s:%d\n",
         (sub == REDISMODULE_SUBEVENT_CLIENT_CHANGE_CONNECTED) ?
             "connection" : "disconnection",
         ci->id,ci->addr,ci->port);
```

Comment:
`ci->id` is a `uint64_t`, you should can't print it with either `%llu` or `%lu`
```suggestion
    printf("Client %s event for client #%llu %s:%d\n",
        (sub == REDISMODULE_SUBEVENT_CLIENT_CHANGE_CONNECTED) ?
            "connection" : "disconnection",
        (unsigned long long)ci->id,ci->addr,ci->port);
```

Reply:
thanks!